src/migration/tasks/RenameUserPropertiesTask.ts

```diff
@@ -0,0 +1,44 @@
+import Constants from '../../utils/Constants';
+import Logging from '../../utils/Logging';
+import MigrationTask from '../MigrationTask';
+import { ServerAction } from '../../types/Server';
+import Tenant from '../../types/Tenant';
+import TenantStorage from '../../storage/mongodb/TenantStorage';
+import User from '../../types/User';
+import global from '../../types/GlobalType';
+
+const MODULE_NAME = 'RenameUserPropertiesTask';
+
+export default class RenameUserPropertiesTask extends MigrationTask {
```

Comment:
Was not necessary to rename this prop.
Last time you did the same with lastSeen has brought more issues than it solved.
Test it carefully!

Reply:
> Was not necessary to rename this prop.

Variables must have name that reflect their purpose. Its purpose have changed so the name.  

> Last time you did the same with lastSeen has brought more issues than it solved.
> Test it carefully!

I hope you also plan to apply that rule to your code changes ;)

Reply:
> > Was not necessary to rename this prop.
> 
> Variables must have name that reflect their purpose. Its purpose have changed so the name.

Except if the impact of this renaming is vast compared to the added value of the renaming which is the case here.

Reply:
> > Last time you did the same with lastSeen has brought more issues than it solved.
> > Test it carefully!
> 
> I hope you also plan to apply that rule to your code changes ;)

It's your code which is on the table here.
Test carefully.

Reply:
> > > Was not necessary to rename this prop.
> > 
> > 
> > Variables must have name that reflect their purpose. Its purpose have changed so the name.
> 
> Except if the impact of this renaming is vast compared to the added value of the renaming which is the case here.

In order to produce maintainable and readable code for the years to come, that rule must be followed. 
I can't imagine how the glibc and Linux kernel code could would have been if that rule would not have been strictly followed. 
And that's where versioning come into: a change have been done that impact an API used externally.
POSIX version IOTCL, REST requires idempotent versioned endpoints (OCPI include 3 digits versioning in endpoints URI, etc.), etc. 
Changes have been made to the E-Mobility REST API that breaks the idempotence of some endpoints  because no changes to the REST API version where done. The REST API code should handle easily version bump on the REST API. That's not the case actually and that's easy to add: 
	+ add one digit to the version to differentiate between minor and major changes
    + add the version as an argument to method handling REST endpoints requests to format the output according to the version
    + a bit of code factoring to have endpoints common code for endpoints that have not change on a version bump